prime-router/src/main/kotlin/cli/LookupTableCommands.kt

```diff
@@ -827,9 +861,28 @@ class LookupTableLoadAllCommand : GenericLookupTableCommand(
         } catch (e: NoSuchFileException) {
             error("Directory ${dir.absolutePath} does not exist")
         }
+
+        // If there is an existing active version then present a diff.
+        val activeTableList = try {
+            tableUtil.fetchList()
+        } catch (e: IOException) {
+            throw PrintMessage("Error fetching the list of tables: ${e.message}", true)
+        }
+
         TermUi.echo("Loading ${files.size} tables from ${dir.absolutePath}...")
-        files.forEach {
+        files.forEach { it ->
+            val tableSha256 = it.Sha256()
             val tableName = it.nameWithoutExtension
+
+            // Check to see if the table is up-to-date
+            val upToDateTable = activeTableList.firstOrNull {
```

Comment:
If you are going to check for the checksum to see if we need to upload a new table then remove the old check for the last modified time above.

Reply:
Removed.  Thanks